vue.config.js

```diff
@@ -73,24 +73,6 @@ module.exports = {
       .use('file-loader')
       .loader('file-loader')
       .end()
-    config.module
-      .rule('monaco-safari-fix')
```

Comment:
This was a Safari fix introduced for monaco editor, which I believe is not required anymore (was fixed upstream in the monaco components).